microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -1,436 +1,163 @@
-  * [CoreWebView2Color Struct](/dotnet/api/microsoft.web.webview2.core.corewebview2color)
-
-##### [Win32/C++](#tab/win32cpp)
-
-* [ICoreWebView2Controller::DefaultBackgroundColor property (get](/microsoft-edge/webview2/reference/win32/icorewebview2controller2#get_defaultbackgroundcolor), [put)](/microsoft-edge/webview2/reference/win32/icorewebview2controller2#put_defaultbackgroundcolor)
-  * [COREWEBVIEW2_COLOR struct](/microsoft-edge/webview2/reference/win32/icorewebview2#corewebview2_color)
+<!-- ------------------------------ -->
+#### APIs for Window to Visual hosting
 
----
+For a list of APIs that can be used when configuring WebView2 Window to Visual hosting (or for Windowed hosting), see [Rendering WebView2 in non-framework apps](./overview-features-apis.md#rendering-webview2-in-non-framework-apps) in _Overview of WebView2 features and APIs_.
 
 
 <!-- ====================================================================== -->
-## Visual hosting
+## Visual hosting: For more granular control over layout
 
-In visual hosting, content is embedded to a given location on the application. This location must handle how content will scale and behave in an app when the user interacts with the application.  In addition to the window management described for windowed hosting, visual hosting will need the app to manage the composition-based rendering, when it receives any user interactions.  For more information about visual hosting, see [Using the Visual layer in desktop apps](/windows/apps/desktop/modernize/visual-layer-in-desktop-apps).
+When using _Visual hosting_, your host app takes spatial input (such as mouse or touch input) from the user.  Your app sends this input to the WebView2 control.
 
-If your WebView2 app uses visual hosting:
+In Visual hosting, content is embedded to a given location on the application. This location must handle how content will scale and behave in an app when the user interacts with the application.  In addition to the window management described for Windowed hosting, Visual hosting will need the app to manage the composition-based rendering, when it receives any user interactions.  For more information about Visual hosting, see [Using the Visual layer in desktop apps](/windows/apps/desktop/modernize/visual-layer-in-desktop-apps).
 
-* Inputs are routed to the app's `HWND` and must be configured to send the spatial input (for example, mouse, touch, and pen) based on positions, _not_ what currently has focus, such as a keyboard.
-
-The following APIs can be used when configuring WebView2 in a visual hosting environment:
+If your WebView2 app uses Visual hosting, inputs are routed to the app's `HWND` and must be configured to send the spatial input (such as mouse, touch, or pen) based on positions, _not_ based on what currently has focus, such as a keyboard.
```

Comment:
```suggestion
If your WebView2 app uses Visual hosting, spatial inputs (such as mouse, touch, or pen) are sent to the app's `HWND`, and not to WebView2 directly. The app should forward this spatial input to the WebView2 if the position of the input is over the WebView2, _not_ based on what currently has focus.
```

Reply:
copypasted but used:
are sent to the app's `HWND`, not sent directly to the WebView2.